consensus/src/dag/rb_handler.rs

```diff
@@ -92,18 +88,21 @@ impl NodeBroadcastHandler {
             .cloned()
             .collect();
         if !missing_parents.is_empty() {
-            // For each missing parent, verify their signatures and voting power
+            // For each missing parent, verify their signatures and voting power. Otherwise, a malicious node can send 
+            // bad nodes with fake parents and cause this peer to issue unnecessary fetch requests.
             ensure!(
                 missing_parents
                     .iter()
                     .all(|parent| { parent.verify(&self.epoch_state.verifier).is_ok() }),
```

Comment:
Should this be in node.verify()?

Reply:
This is to only verify the missing parents. Ideally, we won't miss any parents in the normal case and don't have to verify at all.